src/lib/data.ts

```diff
@@ -70,7 +84,7 @@ const getAvailableDomains = (): Domain[] => {
 }
 
 const getAccessTokens = (): AccessToken[] => {
-  const initialData = getData('accessToken') as AccessToken[] | undefined
+  const initialData = getData('accessTokens') as AccessToken[] | undefined
```

Comment:
I think maybe we should make `getData` have a return type that includes `undefined` we don't need to write `as AccessToken[] | undefined`.
We can make getData like this
```
const getData = <T extends keyof DB>(table: T): DB[T] | undefined => {
```
and then we can probably remove the `as AccessToken[] | undefined` here and still have type safety